aten/src/ATen/native/cuda/PowKernel.cu

```diff
@@ -140,13 +109,9 @@ void pow_tensor_scalar_kernel_impl(TensorIteratorBase& iter,
       return base * base * base;
     });
   } else if (d_exp == -0.5) {
-    gpu_kernel(iter, [=]GPU_LAMBDA(Base_type base) -> Base_type {
-      return invsqrt_(base);
-    });
+    rsqrt_stub(kCUDA, iter);
```

Comment:
currently rsqrt_stub doesn't support BFloat16, yet pow does, so we should extend bfloat16 support to bfloat16 too
Also, stubs would go through dispatch macro again, even though at this point we already know the dtype. We should refactor the code to avoid that.